### ISSUE TITLE
Wish you have a package interface

### ISSUE THREAD

**Issue by mvfki** [yunguan-wang/Spacia#57]
It would be really preferrable if the method is presented as a module, because people really have various input format. 

As for my case, I got 10X Visium HD data which is 8µm-binned into ~300k spots and there are ~20k genes. I really don't want to convert the data into a "counts.txt," which is going to be time-consuming and, more importantly, memory/storage-unfriendly. It will be much appreciated if you can refactor the code to support an AnnData object stored in H5AD file, or other commonly seen data container in R. Additionally, using a sparse matrix instead of letting a pandas DataFrame to hold the densed information will be much appreciated.

**Comment by yunguan-wang** [comment]
You are right and probably shouldn't. Spacia is really only biologically applicable to single cell based SRT. For Visium HD, since the bins will more often than not containing more than one cells, any CCC predicted can be confounded by the co-expression of genes within the same cell.
